src/page/Login2.tsx

```
@@ -0,0 +1,254 @@
+import { useState } from 'react';
+import { Map, Mail, Lock, Eye, EyeOff } from 'lucide-react';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)